{
	"name": "CVE-2020-14422",
	"seq": "2020-14422",
	"type": "CAN",
	"status": "Candidate",
	"phase": {
		"text": "Assigned",
		"date": "20200618"
	},
	"desc": "Lib/ipaddress.py in Python through 3.8.3 improperly computes hash values in the IPv4Interface and IPv6Interface classes, which might allow a remote attacker to cause a denial of service if an application is affected by the performance of a dictionary containing IPv4Interface or IPv6Interface objects, and this attacker can cause many dictionary entries to be created. This is fixed in: v3.5.10, v3.5.10rc1; v3.6.12; v3.7.9; v3.8.4, v3.8.4rc1, v3.8.5, v3.8.6, v3.8.6rc1; v3.9.0, v3.9.0b4, v3.9.0b5, v3.9.0rc1, v3.9.0rc2.",
	"refs": [
		{
			"text": "https://security.netapp.com/advisory/ntap-20200724-0004/",
			"source": "CONFIRM",
			"url": "https://security.netapp.com/advisory/ntap-20200724-0004/"
		},
		{
			"text": "FEDORA-2020-1ddd5273d6",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YILCHHTNLH4GG4GSQBX2MZRKZBXOLCKE/"
		},
		{
			"text": "FEDORA-2020-705c6ea5be",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FCCZTAYZATTNSNEAXWA7U3HCO2OVQKT5/"
		},
		{
			"text": "FEDORA-2020-87c0a0a52d",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NTBKKOLFFNHG6CM4ACDX4APHSD5ZX5N4/"
		},
		{
			"text": "FEDORA-2020-982b2950db",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/CTUNTBJ3POHONQOTLEZC46POCIYYTAKZ/"
		},
		{
			"text": "FEDORA-2020-b513391ca8",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/X36Y523UAZY5QFXZAAORNFY63HLBWX7N/"
		},
		{
			"text": "FEDORA-2020-bb919e575e",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LE4O3PNDNNOMSKHNUKZKD3NGHIFUFDPX/"
		},
		{
			"text": "FEDORA-2020-c3b07cc5c9",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/V3TALOUBYU2MQD4BPLRTDQUMBKGCAXUA/"
		},
		{
			"text": "FEDORA-2020-c539babb0a",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/36XI3EEQNMHGOZEI63Y7UV6XZRELYEAU/"
		},
		{
			"text": "FEDORA-2020-d30881c970",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OXI72HIHMXCQFWTULUXDG7VDA2BCYL4Y/"
		},
		{
			"text": "FEDORA-2020-d808fdd597",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/V53P2YOLEQH4J7S5QHXMKMZYFTVVMTMO/"
		},
		{
			"text": "FEDORA-2020-dfb11916cc",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/VT4AF72TJ2XNIKCR4WEBR7URBJJ4YZRD/"
		},
		{
			"text": "FEDORA-2020-efb908b6a8",
			"source": "FEDORA",
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/CNHPQGSP2YM3JAUD2VAMPXTIUQTZ2M2U/"
		},
		{
			"text": "GLSA-202008-01",
			"source": "GENTOO",
			"url": "https://security.gentoo.org/glsa/202008-01"
		},
		{
			"text": "https://bugs.python.org/issue41004",
			"source": "MISC",
			"url": "https://bugs.python.org/issue41004"
		},
		{
			"text": "https://github.com/python/cpython/pull/20956",
			"source": "MISC",
			"url": "https://github.com/python/cpython/pull/20956"
		},
		{
			"text": "https://www.oracle.com/security-alerts/cpujan2021.html",
			"source": "MISC",
			"url": "https://www.oracle.com/security-alerts/cpujan2021.html"
		},
		{
			"text": "[debian-lts-announce] 20200715 [SECURITY] [DLA 2280-1] python3.5 security update",
			"source": "MLIST",
			"url": "https://lists.debian.org/debian-lts-announce/2020/07/msg00011.html"
		},
		{
			"text": "[debian-lts-announce] 20230515 [SECURITY] [DLA 3424-1] python-ipaddress security update",
			"source": "MLIST",
			"url": "https://lists.debian.org/debian-lts-announce/2023/05/msg00016.html"
		},
		{
			"text": "openSUSE-SU-2020:0931",
			"source": "SUSE",
			"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00003.html"
		},
		{
			"text": "openSUSE-SU-2020:0940",
			"source": "SUSE",
			"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00006.html"
		},
		{
			"text": "openSUSE-SU-2020:0989",
			"source": "SUSE",
			"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00032.html"
		},
		{
			"text": "openSUSE-SU-2020:1002",
			"source": "SUSE",
			"url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00041.html"
		},
		{
			"text": "USN-4428-1",
			"source": "UBUNTU",
			"url": "https://usn.ubuntu.com/4428-1/"
		}
	],
	"votes": {}
}
